src/pages/index.js

```diff
@@ -7,41 +7,75 @@ import Container from '../components/Container'
 import { useSiteMetadata } from '../hooks/use-site-metadata'
 import Intro from '../components/Intro'
 import ConferencesList from '../components/ConferencesList'
+import ConferencesSearch from '../components/ConferencesSearch'
+import EmptyListText from '../components/EmptyListText'
 
 const URL = 'https://homeferences.github.io/list/homeferences.json'
 
 const IndexPage = () => {
   const [conferences, setConferences] = useState([])
+  const [loading, setLoading] = useState(true)
   const { intro } = useSiteMetadata()
 
-  useEffect(() => {
-    const fetchData = async () => {
-      try {
-        const response = await fetch(URL)
-        let data = await response.json()
-
-        setConferences(data)
-      } catch (e) {
-        console.log('error', e)
-      }
+  const fetchData = async (query = '') => {
+    try {
+      setLoading(true)
+      const response = await fetch(URL)
+      let data = await response.json()
+
+      setConferences(
+        data.filter(conference => {
+          let inDescription = conference.description
+            ? conference.description
+                .toLowerCase()
+                .indexOf(query.toLowerCase()) >= 0
+            : false
+          let inTopic = conference.topic
+            ? conference.topic.toLowerCase().indexOf(query.toLowerCase()) >= 0
+            : false
+          let inTitle = conference.name
+            ? conference.name.toLowerCase().indexOf(query.toLowerCase()) >= 0
+            : false
+          return inDescription || inTopic || inTitle
+        })
+      )
+      setLoading(false)
+    } catch (e) {
+      console.log('error', e)
```

Comment:
Please remove logs (only for development)

Reply:
It was already there, but i can remove it